viewing_party/party.py

```diff
@@ -1,23 +1,153 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    if not title or not genre or not rating: 
+        return None
+    if title and genre and rating:
+        movie_dict = {}
+        movie_dict['title'] = title
+        movie_dict['genre'] = genre
+        movie_dict['rating'] = rating
+
+        return movie_dict

```

Comment:
If we've passed the guard clause, then we know that title/genre/rating have valid values and we do not need to check for this condition on line 6. Instead, would prefer for lines 7-12 to be outside the conditional and indented at the main level of the method.

A note about guard clauses - The Guard Clause pattern helps in these cases by “inverting” that way of thinking and rapidly returning false conditions and keeping the important code blocks (usually the longest) in the lowest indentation level possible. If title/genre/rating are valid, then we don't need to have the rest of logic indented in an else statement. We can unindent them and have them in the main body of the function. 

You can read more about guard clauses [here](https://medium.com/lemon-code/guard-clauses-3bc0cd96a2d3)
------------------

You can also create your dictionary with literal syntax and return it without first creating the variable movie_dict:
```python
if not title or not genre or not rating: 
    return None 
return { "title": title, "genre": genre, "rating": rating }